ShoeStore/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "pizza",
```

Comment:
like this

Reply:
No bro this is pure HTML CSS js based work. I am sorry just noticed that I pushed a few changes that I made in the shoe store website also.

Reply:
I just made a push and removed the react based website. Could you please check it once noq